src/components/templates/PartyMap/components/Map/Map.tsx

```diff
@@ -1,36 +1,31 @@
 import React, { useMemo } from "react";
+import { useWindowSize } from "react-use";
 
 import { DEFAULT_MAP_BACKGROUND } from "settings";
 
 import { RefiAuthUser } from "types/fire";
 import { Room } from "types/rooms";
+import { Dimensions, Position } from "types/utility";
 import { PartyMapVenue } from "types/venues";
 
 import { useValidImage } from "hooks/useCheckImage";
 
 import { MapRoom } from "./MapRoom";
 
 import styles from "./Map.module.scss";
-
-interface MapTitleProps {
-  title: string;
-}
-
-const MapTitle: React.FC<MapTitleProps> = ({ title }) => {
-  const titleParts = useMemo(
-    () => title.split(" ").map((part, idx) => <span key={idx}>{part}</span>),
-    [title]
-  );
-  return <h1 className={styles.MapTitle}>{titleParts}</h1>;
-};
-
 interface PortalsProps {
   portals: Room[];
   space: PartyMapVenue;
   selectPortal: (room: Room) => void;
+  safeZoneBounds: Dimensions & Position;
 }
 
-const Portals: React.FC<PortalsProps> = ({ space, selectPortal, portals }) => {
+const Portals: React.FC<PortalsProps> = ({
+  space,
+  selectPortal,
+  portals,
+  safeZoneBounds,
```

Comment:
[NOTE] There might be some other way to avoid passing the same prop three components deep, but then, probably it's clear enough by their names and location that these components are meant to only work together